packages/merge-styles/src/IRawStyleBase.ts

```diff
@@ -335,6 +335,18 @@ export interface IRawStyleBase extends IRawFontStyle {
    */
   appearance?: ICSSRule | string;
 
+  /**
+   * Lets you apply graphical effects such as blurring or color shifting to the area
+   * behind an element. Because it applies to everything behind the element, to see
+   * the effect you must make the element or its background at least partially transparent.
+   */
+  backdropFilter?: ICSSRule | string;
+
+  /**
+   * Edge requires the -webkit prefix backdrop-filter.
+   */
+  WebkitBackdropFilter?: ICSSRule | string;
```

Comment:
is this "W" capitol for a reason?

Reply:
okay, the W does appear to be convention. But I believe we don't need to specify standard autoPrefix rules. Only prefix rules that are unique to the browser.

Reply:
hmm @micahgodbolt we "autoprefix" very very few things, and it is solely managed by merge-styles, so unless the prefixing logic handles the conversion, yes you have to do this.

Sure is weird that Edge would require a -webkit prefix, rather than -ms!

Reply:
@joetheday do you have any background on why Edge needs the webkit prefix? And if it IS required, should we add it to the list of items we DO autoprefix? Seems silly to write both.

Reply:
I do have that background, and we can follow up offline. But for now, yes the prefix is required. 

Will autoprefix handle it if we remove it here?